README.rst

```diff
@@ -172,35 +187,31 @@ Then a common command will work.
 
 Note that OpenMP and BLAS are disabled on this platform.
 
-CMake support
-=============
-
-One can also build the plugin using CMake:
 
-.. code-block:: console
+Testing
+=======
 
-    $ cmake -S. -B build
-    $ cmake --build build
+To test that a plugin is working correctly, 
 
-Supported options are
+Python Test
+^^^^^^^^^^^ 
 
-- ``-DENABLE_WARNINGS:BOOL=ON``
-- ``-DENABLE_NATIVE:BOOL=ON`` (for ``-march=native``)
-- ``-DENABLE_BLAS:BOOL=ON``
-- ``-DENABLE_OPENMP:BOOL=ON``
-- ``-DENABLE_CLANG_TIDY:BOOL=ON``
+Test the Python code with:
 
-Testing
-=======
+.. code-block:: console
 
-To test that a plugin is working correctly, test the Python code with:
+    $ make test-python device=${PL.DEVICE}
```

Comment:
I find the prompt `$` annoying in code blocks since it gets in the way of copy-pasting. Should we not remove it?

Reply:
Yes, please remove `$` as the "console" code-block conveys the use case of these shell commands.

Reply:
Ok but Do I remove only the new ones or all the `$` symbols in the console blocks? :thinking: